app/javascript/.storybook/main.js

```diff
@@ -68,6 +68,8 @@ module.exports = {
         '@crayons': path.resolve(__dirname, '../crayons'),
         '@utilities': path.resolve(__dirname, '../utilities'),
         '@components': path.resolve(__dirname, '../shared/components'),
+        react: 'preact/compat',
```

Comment:
We talked about it already, but just a note @aitchiss to add this same configuration to environment.js once the component is being consumed in the application.

https://github.com/forem/forem/blob/master/config/webpack/environment.js#L21-L29